lib/phoenix_live_view/channel.ex

```diff
@@ -595,6 +598,19 @@ defmodule Phoenix.LiveView.Channel do
     {:stop, :shutdown, :no_session}
   end
 
+  defp check_version_match(params) do
+    server_version = Keyword.get(project(), :version, :not_found)
```

Comment:
It is preferable to this:

```suggestion
    server_version = Application.spec(:phoenix_live_view :vsn) |> List.to_string()
```

As Mix is not available during a release. :) Although we should decide if we want to use the package version or a separate one.

Reply:
I didn't know that, thanks for pointing it out. When talking about this separate versioning, would it be in a file that is consumed both by the server code and the client one, or also set in two different places?